cylc/flow/etc/examples/event-driven-cycling/bin/trigger

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -eu
+
+if [[ $# -lt 1 ]]; then
+    echo 'Usage ./trigger WORKFLOW_ID [KEY=VALUE ...]' >&2
+    echo
+    echo 'Trigger a new cycle in the target workflow.'
+    echo 'Any environment variable KEY=VALUE pairs will be broadcasted to'
+    echo 'all tasks in the cycle.'
+    exit 1
+fi
+
+# determine the workflow
+WORKFLOW_ID="$1"
+shift
+WORKFLOW_RUN_DIR="${HOME}/cylc-run/${WORKFLOW_ID}"
+EXT_TRIGGER_DIR="${WORKFLOW_RUN_DIR}/triggers"
+mkdir -p "$EXT_TRIGGER_DIR"
+
+# pick a trigger-id
+TRIGGER_ID="$(date --iso-8601=seconds)"
```

Comment:
This is in extended format with `:` in the name. Seeing as it is used in a filepath, is that a problem?